### ISSUE TITLE
Is it by designed that report is not generated when all class tests have @ignore ?

### ISSUE THREAD

**Issue by AnthonyTonda** [renatoathaydes/spock-reports#242]
Given all classes have @ignore
When I run these classes
Then spock report generates no report at all
But I would expect to have the generated report where I see the list of tests didn’t run

Spock Version: 1.3-groovy-2.4
Spock Report Version: 1.2.7

**Comment by renatoathaydes** [comment]
I agree, and am surprised that's the case! If you would like to submit a pull request, this should be easy to fix.

Otherwise, I may have a look within a couple of months, given I am too busy at the moment with other things.

**Comment by AnthonyTonda** [comment]
Hi Renato, will try to raise a PR ;) to which branch I should select for raising the PR ? to dev ?

**Comment by renatoathaydes** [comment]
Yes, `dev`, I've just updated that!

**Comment by renatoathaydes** [comment]
I tried to fix this issue, but it seems impossible because Spock is not even calling spock-reports' extension when the spec is fully ignored.

That looks like a bug in Spock, because extensions have a method, which I've implemented, for this case:

```groovy
    @Override
    void specSkipped( SpecInfo spec )
```

**Comment by renatoathaydes** [comment]
The bug is fixed on the Spock side, just waiting for the PR to be merged and the fix released.

**Comment by renatoathaydes** [comment]
This was fixed by Spock long ago and I have enabled the test checking for this now.
